packages/nocodb/src/models/Filter.ts

```
@@ -690,6 +690,32 @@ export default class Filter implements FilterType {
+        context.base_id,
+        MetaTable.FILTER_EXP,
+        {
+          // @ts-ignore
+          condition: { fk_hook_id: viewId },
+        },
+      );
+      // @ts-ignore
+      await NocoCache.setList(CacheScope.FILTER_EXP, [viewId], filterObjs);
+    }
+
+    return filterObjs?.map((f) => this.castType(f)) || [];
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)